Marlin/src/lcd/extui/lib/dgus/mks/DGUSScreenHandler.cpp

```diff
@@ -1251,6 +1251,12 @@ void DGUSScreenHandler::MKS_FilamentLoadUnload(DGUS_VP_Variable &var, void *val_
       queue.enqueue_one_now(buf);
       queue.enqueue_now_P(PSTR("G90"));
     }
+  #else
+    queue.enqueue_now_P(PSTR("T1"));
+    queue.enqueue_now_P(PSTR("G91"));
+    snprintf_P(buf, 40, PSTR("G1 E%d F%d"), (int)distanceFilament * filamentDir, FilamentSpeed * 60);
+    queue.enqueue_one_now(buf);
+    queue.enqueue_now_P(PSTR("G90"));
```

Comment:
Note that `enqueue_now_P` and `enqueue_one_now` can just go into an infinite loop if the buffer is full, so this is not a very good way to do this. I bet you can get Marlin to lock up by poking the screen enough. I would advised that the coders use a different technique for this kind of thing. Reducing these actions down to a single (custom) G-code would be wiser than trying to stuff the buffer this way.